templates/feedback-en.html

```diff
@@ -8,8 +8,8 @@
 	"breadcrumb": [
 		{ "title": "Canada.ca", "link": "https://www.canada.ca/en.html" }
 	],
-	"css": ["demos/feedback/demo/feedback"],
-	"js": ["demos/feedback/demo/feedback"],
+	"css": ["https://wet-boew.github.io/v4.0-ci/demos/feedback/demo/feedback.min.css"],
+	"js": ["https://wet-boew.github.io/v4.0-ci/demos/feedback/demo/feedback.min.js"],
```

Comment:
FYI the supporting JS file isn't getting injected into the generated version of this page template.

So GCWeb's version of the feedback page template will still be broken even with these fixes.

Reply:
Just fixed it. Renamed ``js`` to ``script`` after spotting it in ``sites/resources-inc/footer.html`` and some méli-mélo pages.